solarforecastarbiter/tests/test_datamodel.py

```diff
@@ -851,3 +851,12 @@ def test_EventForecast_parameters(site_metadata, interval_label, variable):
 
     assert fx.interval_label == "event"
     assert fx.variable == "event"
+
+
+@pytest.mark.parametrize('model', [
+    datamodel.Forecast,
+    datamodel.EventForecast,
+])
+def test_event_forecast_from_dict(model, single_event_forecast):
+    event_fx_dict = single_event_forecast.to_dict()
+    assert isinstance(model.from_dict(event_fx_dict), datamodel.EventForecast)
```

Comment:
can you also add a test for loading a raw report w/ event forecast since that is where the original issue stems from